src/debugger/configuration-provider.ts

```diff
@@ -66,7 +66,7 @@ async function resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefi
             return;
         }
 
-        const installInputs: InstallInputs = { parameters: parameters.value, credentialSet: credentialSet.value };
+        const installInputs: InstallInputs = { namespace: "", parameters: parameters.value, credentialSet: credentialSet.value };
```

Comment:
💡☕️ Just a note: Please correct me, I presume this starts with `namespace` as empty string and not any default value because later this gets populated and used according to the use case.

Reply:
Good question. I was just focusing on getting the porter panel working again but wasn't doing everything necessary to support allowing the user to specify the namespace (#53). 

The extension can now list/get all installations in the entire porter database, not just in the currently configured namespace in your ~/.porter/config.yaml.

When you create an installation using the Porter Install command in the VS Code extension, I am setting the namespace to "", but that is actually overridden by whatever is configured in your config file as well. So if your current namespace is "mystuff", and you use the extension to install something, it will be installed into mystuff.

Let me remove setting this field on this line since that's unclear, and leave a comment explaining that the config file will determine the namespace instead.

Long term, it would be nice to have the Porter Install command in the extension prompt for the namespace.